spec/publisher/removing_content_with_redirect_spec.rb

```diff
@@ -0,0 +1,59 @@
+feature "Removing content by redirecting it from Publisher", publisher: true do
```

Comment:
On commit message for this it's probably worth tweaking:

```
We confirmed this is the correct behaviour with the mainstream team.
```

to: 

```
As far as we've been able to determine this is correct and intended behaviour, however it is somewhat odd and surprising.
```

As it's not been officially confirmed and wouldn't want to mislead anyone with this assertion.